impl/src/main/java/org/ehcache/internal/persistence/DefaultLocalPersistenceService.java

```diff
@@ -77,12 +79,13 @@ public synchronized void stop() {
         // passes on windows
         rw.close();
         if (!lockFile.delete()) {
-          // todo log something?;
+          LOGGER.info("Lock file was not deleted {}.", lockFile.getPath());
```

Comment:
@lorban and I looked into this a bit closer. Looks like failing to delete the file will not prevent getting a lock when starting. Thus the log level could even stay at DEBUG.